app/events/inventory_error.rb

```diff
@@ -1,17 +1,9 @@
 class InventoryError < StandardError
   # @return [Event]
   attr_accessor :event
-  # @return [Integer]
-  attr_accessor :item_id
-  # @return [Integer]
-  attr_accessor :storage_location_id
 
   # @param message [String]
-  # @param item_id [Integer]
-  # @param storage_location_id [Integer]
-  def initialize(message, item_id, storage_location_id)
-    super(message)
-    self.item_id = item_id
-    self.storage_location_id = storage_location_id
+  def initialize(message)
```

Comment:
You can probably just remove this definition entirely since it does nothing but call `super`.